ui/src/api/__tests__/client.test.ts

```diff
@@ -133,7 +133,7 @@ describe("api client", () => {
 		it("should reload current page", () => {
 			// mock window.location.reload
 			const globalWindow = global.window;
-			global.window = Object.create(window);
+			global.window ??= Object.create(window);
```

Comment:
workaround to suppress the error, `TypeError: Cannot redefine property: window` appearing after NodeJS 18.16 upgrade. this looks like it's a bug that has been fixed upstream in NodeJS 20 but hasn't been back-ported to v18 yet.

Reply:
I wasn't planning to upgrade to NJS 20 until it's LTS ~ Nov 2023.
I was hoping that the bug fix for this would be back-ported to 18.x, but there hasn't been a release to 18.x since 18.16.0 on 12 Apr